includes/class-wc-admin-loader.php

```diff
@@ -145,7 +163,7 @@ public static function load_features() {
 	/**
 	 * Registers a basic page handler for the app entry point.
 	 *
-	 * @todo The entry point for the embed needs moved to this class as well.
+	 * TODO: The entry point for the embed needs moved to this class as well.
```

Comment:
In #1474 we converted all `TODO:` and similar comments to `@todo`. Any reason to change it back? :smile:

Reply:
phpcs moans about them :)